crates/http/src/trigger.rs

```diff
@@ -2,13 +2,11 @@ use serde::{Deserialize, Serialize};
 use spin_locked_app::MetadataKey;
 
 /// Http trigger metadata key
-pub const METADATA_KEY: MetadataKey<Metadata> = MetadataKey::new("trigger");
+pub const METADATA_KEY: MetadataKey<Metadata> = MetadataKey::new("triggers.http");
```

Comment:
Ditto re: backward compat.

Reply:
This isn't used anymore in my pr, bad leftover. If we can wrap up above I can remove this one.